src/core/Akka.Tests/Actor/AskTimeoutSpec.cs

```diff
@@ -56,16 +56,24 @@ public AskTimeoutSpec()
             var actor = Sys.ActorOf<SleepyActor>();
 
             var actorCell = actor as ActorRefWithCell;
+            Assert.NotNull(actorCell);
```

Comment:
Mostly to make ReSharper compiler warnings go away / throw the correct assertion error in the event that the underlying `ActorCell` is modified in a meaningful way in the future.